package.json

```diff
@@ -20,5 +20,32 @@
     "youtube",
     "video",
     "thumbnail"
-  ]
+  ],
+  "scripts": {
+    "test": "jest",
+    "test:watch": "jest --watch"
+  },
+  "jest": {
+    "automock": false,
+    "globals": {
+      "__DEV__": true
+    },
+    "modulePathIgnorePatterns": [
+      "<rootDir>/example/*"
+    ],
+    "modulePaths": [
+      "<rootDir>/src/**/*.test.js"
+    ],
+    "preset": "react-native"
+  },
+  "devDependencies": {
+    "babel-jest": "^21.2.0",
+    "babel-preset-react-native": "^4.0.0",
+    "jest": "^21.2.1",
+    "react-test-renderer": "^16.1.1"
+  },
+  "dependencies": {
+    "react": "^16.1.1",
```

Comment:
`react` & `react-native` should be peer dependencies otherwise it would install two versions because of the library.

Reply:
🤦‍♂️  On it.

What is the minimum version allowed? For testing, we do need both around as well.

Reply:
The package doesn't use any native stuff so I guess it should work well on any of the newest versions, I think we can keep a peerDep of the latest version, what's your opinion?